### DISCUSSION TITLE
Error while load website

### DISCUSSION THREAD

**Discussion by rquevedo** [Tecnativa/doodba#419] [Q&A]
Hi folks,
I'm having an issue in some deploys(not all).  While website is loading some request fails and provokes that website editor get unusable. The error:

```bash
odoo_1    | 2021-05-02 20:35:44,163 50 INFO prod odoo.http: <function DataSet.call_kw at 0x7ff692c87f28>, /web/dataset/call_kw: Function declared as capable of handling request of type 'json' but called with a request of type 'http' 
odoo_1    | 2021-05-02 20:35:44,164 50 INFO prod werkzeug: 66.232.110.3 - - [02/May/2021 20:35:44] "GET /web/dataset/call_kw HTTP/1.1" 400 - 1 0.000 0.004
odoo_1    | 2021-05-02 20:35:44,723 51 INFO prod odoo.http: <function DataSet.call_kw at 0x7ff692c87f28>, /web/dataset/call_kw: Function declared as capable of handling request of type 'json' but called with a request of type 'http' 
```
In network tab we can see that fail requests was launched by others that ends in redirect:

![Captura de pantalla de 2021-05-02 16-38-59](https://user-images.githubusercontent.com/2567590/116827143-94b70100-ab65-11eb-9359-383ce65fea6d.png)

Seems like deploy issue but i don't know where start to look.
Any help will be appreciated .

**Comment by pedrobaeza** [comment]
This has nothing to do with doodba itself, so please abstain on polluting about other Odoo stuff here.

**Reply by rquevedo** [reply]
> This has nothing to do with doodba itself, so please abstain on polluting about other Odoo stuff here.

I think that this "maybe" is a configuration issue in doodba or at least in traefik, because i can not reproduce the error with same database, filestore and addons deployed without doodba.

In another topic, I only made the question in case someone in the community had the same problem. Do not forget that if we can solve it, it can save time in the future to all of us. Ultimately, that is the goal of the communities. I don't think it's pollution, but it's just my opinion.

thanks anyway

**Reply by pedrobaeza** [reply]
This is not a general Odoo forum, but Doodba one. The problem is probably due to a crawler consuming services in a bad way, and Odoo logging it. Nothing even in Odoo side.
